src/main/java/seedu/address/model/book/Book.java

```diff
@@ -0,0 +1,111 @@
+package seedu.address.model.book;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.genre.Genre;
+
+/**
+ * Represents a Person in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Book {
+
+    // Identity fields
+    private final Title title;
+    private final SerialNumber serialNumber;
+    private final Author author;
+    private final Set<Genre> genres = new HashSet<>();
+
```

Comment:
We need an additional Loan field to know whether this book is loaned, available or overdue

Reply:
Will integrate it into the book class once it is done!

Reply:
Will integrate it once it is created!